browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -455,9 +455,12 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_CONTINUE_TO_WALLET" desc="Text for continue to wallet button">
         Continue to wallet
       </message>
-      <message name="IDS_MENU_SET_DEFAULT_BROWSER" desc="Title for set default browser menu item">
+      <message name="IDS_SET_DEFAULT_BROWSER" desc="Title for set default browser menu item">
         Set as default browser
       </message>
+      <message name="IDS_MENU_SET_DEFAULT_BROWSER" desc="Title for set default browser menu item">
```

Comment:
It has same desc as previous strings, are these 2 the same?

Reply:
changed desc